src/interfaces/assistants_web/src/components/Shared/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,72 @@
+import { Fieldset, Input } from '@headlessui/react';
+import { useState } from 'react';
+
+import { Icon } from '@/components/Shared/Icon';
+import { STYLE_LEVEL_TO_CLASSES } from '@/components/Shared/Text';
+import { cn } from '@/utils';
+
+type Props = {
```

Comment:
we could extend the input field props and avoid adding max length, classname, placeholder, value, etc